test/test_utils.py

```diff
@@ -889,21 +888,43 @@ def f():
         self.assertEqual(r1.device.type, 'meta')
         self.assertEqual(r2.device.type, 'meta')
 
-
     def test_nn_module(self):
         with torch.device('meta'):
             m = nn.Linear(40, 50)
         self.assertEqual(m.weight.device.type, 'meta')
 
     def test_set_default_device(self):
         try:
-            set_default_device('meta')
+            torch.set_default_device('meta')
             r = torch.empty(2, 2)
         finally:
-            set_default_device(None)
+            torch.set_default_device(None)
 
         self.assertEqual(r.device.type, 'meta')
 
+    def test_get_default_device(self):
+        torch.set_default_device('meta')
+        self.assertEqual(torch.get_default_device().type, 'meta')
+        torch.set_default_device(None)
+
+    @unittest.skipIf(
+        not (torch.cuda.is_available() and torch.cuda.device_count() > 1),
```

Comment:
Please use `TEST_MULTIGPU` from `torch.testing._internal.common_cuda`
```suggestion
        not TEST_MULTIGPU,
```

Reply:
Done :D